### PR TITLE
Improve telemetry docs & increase bit field to 16 bits

### PR DESCRIPTION
**Pull request by matt-FFFFFF**
## Overview/Summary

* Improve telemetry docs
* Increase length of bit field to 16 bits

## Evidence

image

### CODE REVIEW

**Review by jtracey93 — CHANGES_REQUESTED** [review]
Nice work @matt-FFFFFF,

A few things for considerations and adding if you agree, alongside code comments:

1. Can we add a link to this page where you think its relevant for further reading customers on the CUA feature https://docs.microsoft.com/en-us/azure/marketplace/azure-partner-customer-usage-attribution
2. Should we add a link to some user-friendly explanation of bitfields for transparency
3. Should we add a link to the code where this is implemented to help interested parties understand it for transparency

File: docs/wiki/[Variables]-disable_telemetry.md [hunk 1]
```diff
@@ -38,17 +38,68 @@ None
...
+| Identity | `67becfb7-b296-43a9-ba38-0b5c19cb065a` | `identity` |
+| Management | `6fffb9f9-2691-412a-837e-3f72dcfe70cb` | `management` |
+
+> Note: Identity is currently disabled until we deploy identity resources
+
+### ARM Deployment Naming
+
+The ARM deployment name is constricted as follows:
+
+`pid-<UUID>_<module_version>_<bitfield>_<random_id>`
```
[the remark this answers is not in the document]
Comment (matt-FFFFFF) [reply docs/wiki/[Variables]-disable_telemetry.md:62]: Done below

**Review by jtracey93 — APPROVED** [review]

**Review by krowlandson — APPROVED** [review]
LGTM 🥳

**Pull request merged**